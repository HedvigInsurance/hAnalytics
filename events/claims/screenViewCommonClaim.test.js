const validateEvent = require("../../commons/validateEvent.js")

test('validate', () => {
    const event = validateEvent(__filename, {
        iconName: "theft",
    })

    expect(event).toMatchSnapshot()
});