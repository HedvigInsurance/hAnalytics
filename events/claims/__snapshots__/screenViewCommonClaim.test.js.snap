// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`validate 1`] = `
Object {
  "name": "screen_view_common_claim_detail",
  "variables": Object {
    "icon_name": "theft",
  },
}
`;
